grakn-graph/src/test/java/ai/grakn/generator/ConceptIds.java

```diff
@@ -32,6 +32,10 @@ public ConceptIds() {
 
     @Override
     public ConceptId generate() {
-        return ConceptId.of(gen(String.class));
+        if (random.nextBoolean()) {
+            return ConceptId.of(gen(String.class));
+        } else {
+            return ConceptId.of("bar");
```

Comment:
It's two reasons: to produce more "real" data (instead of weird unicode characters), and to more frequently produce edge cases (the generator will often produce duplicate concept IDs). This line was what helped me catch a lot of the stranger bugs I found.